test/validate/pass_pointer_exists.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+
+TMP="$(mktemp -d)"
+cleanup() { rm -rf "$TMP"; }
```

Comment:
Minor thing, but other commands call this prelude `clean`. Can you do the same for consistency?